osv/impact.py

```diff
@@ -174,11 +190,31 @@ def _get_affected_range(self,
         if equivalent_fix_commit:
           break
 
-      range_collector.add(equivalent_regress_commit, equivalent_fix_commit)
+      # Get the latest equivalent commit in the last_affected range (if
+      # present).
+      equivalent_last_affected_commit = None
+      if last_affected_commits:
```

Comment:
Do we need this if statement, or is it redundant given the for loop? The `fix_commits` case doesn't have it.